h/views/activity.py

```diff
@@ -154,6 +154,29 @@ def user_annotation_count(aggregation, userid):
         result['stats'] = {
             'annotation_count': group_annotation_count,
         }
+
+        if self.group.type == 'private':
+            result['group_users_display_info'] = {
+                'heading': 'Members',
+                'members': members,
+                'subtitle': 'Invite new members',
+                'share_msg': 'Sharing the link lets people join this group:',
+            }
+        if self.group.type == 'open':
+            result['group_users_display_info'] = {
+                'heading': 'Moderators',
+                'members': moderators,
+                'subtitle': 'Share group',
+                'share_msg': 'Sharing the link lets people view this group:',
+            }
+        if self.group.type == 'restricted':
+            result['group_users_display_info'] = {
+                'heading': 'Contributors',
+                'members': members,
+                'subtitle': 'Share group',
+                'share_msg': 'Sharing the link lets people view this group:',
+            }
+
```

Comment:
As we discussed in person, perhaps consider moving a few of the `group_users_display_info` items into the main `group` dictionary if they fit more with the "group" than the "group's users"...